utils/numbers.py

```diff
@@ -15,9 +18,20 @@ def rounded(
     :param decimals: The amount of decimals to round value
     :param as_string: Whether the value should be returned as Decimal or string.
     For testing, it's useful to be able to parse the Decimal value directly to string.
+    :param round_to_nearest: Rounds the value to the nearest decimal passed.
+    I.e. the nearest 0.05 of 3.1416 -> 3.15
     :return: A rounded Decimal number
     """
-    number = round(Decimal(value), decimals)
+    number = value
```

Comment:
I think we could skip this assignment and just use `value` everywhere. or rename the parameter to `number`

Reply:
I thought so too but I went the other way to avoid overwriting parameters, but I guess it won't make much of a difference here.